doc/ws.md

```diff
@@ -0,0 +1,121 @@
+# Websockets interface
+
+## What is this?
+
+This interface is a new way to talk with Orion using websockets, this is point to improve the comunication with a faster and efficient protocol, now we have a persistent connection with a open socket. 
```

Comment:
Not sure what is meant by "this is point" ...
"this is a way"?

More:  faster and efficient protocol => "faster and **more** efficient protocol"

And: with a open socket  =>  with **an** open socket